Update translation strings

nephros opened the pull request:
Results from:

 -  `git checkout 3.2.4`
 -  `lupdate *pro -ts translations/settings-patchmanager.ts`
 -  `restore translations/settings-patchmanager-*ts`
 - `git diff  > translations-3.2.4.diff`
 -  `git switch master`
 - `git apply  translations-3.2.4.diff`
 - `git commit`
 -  `lupdate *pro -ts translations/settings-patchmanager.ts`
 -  `restore translations/settings-patchmanager-*ts`
 - `git commit`

Should fix #357.

Olf0 reviewed (approved):
Looks fine.

I only asked myself what [the new source string "Advanced" in line 579](https://github.com/sailfishos-patches/patchmanager/pull/360/files#diff-1f485120db4530253da92d081a3d5aff4d0d2e962d01317f8c1ca73010c5a2f9R579) (from [`src/qml/SettingsPage.qml`, line 108](https://github.com/sailfishos-patches/patchmanager/blob/master/src/qml/SettingsPage.qml#L108), introduced last week via MR #334) as a stand-alone word is supposed to mean, usually it shall be "Advanced \<something\>".  If "Advanced settings" are supposed to be addressed, I would rather call them "Expert settings", "Special settings" or "Advanced functions" / "Advanced functionality".

nephros commented:
> Looks fine.
> 
> I only asked myself what [the new source string "Advanced" in line 579](https://github.com/sailfishos-patches/patchmanager/pull/360/files#diff-1f485120db4530253da92d081a3d5aff4d0d2e962d01317f8c1ca73010c5a2f9R579) (from [`src/qml/SettingsPage.qml`, line 108](https://github.com/sailfishos-patches/patchmanager/blob/master/src/qml/SettingsPage.qml#L108), introduced last week via MR #334) as a stand-alone word is supposed to mean, usually it shall be "Advanced <something>". If "Advanced settings" are supposed to be addressed, I would rather call them "Expert settings", "Special settings" or "Advanced functions" / "Advanced functionality".
> 
> IMO this should be fixed first, rsp. in one go with / in this MR.

It's a Section header in the Settings page. The page title is Settings, and the other section header is "General".
As such it should be clear from context that it's referring to (advanced) Settings.

And it's not new, just moved position within that file. It was introduced in #199.

Olf0 commented:
> It's a Section header in the Settings page. The page title is [Settings](https://github.com/sailfishos-patches/patchmanager/blob/master/src/qml/SettingsPage.qml#L79), and the other section header is ["General"](https://github.com/sailfishos-patches/patchmanager/blob/master/src/qml/SettingsPage.qml#L82). As such it should be clear from context that it's referring to (advanced) Settings.

Sorry, I should have turned on my testing phone and have taken a look at it.  That is sure fine.

> And it's not new, just moved position within that file. It was introduced in #199.

Ah, something Git blame does not show so easily.
